### ISSUE TITLE
Add support for formulas/equations from markdown

### ISSUE THREAD

**Issue by LucAngevare** [jgm/pandoc#6721]
Right now I'm having the issue that I've written some formulas down in **GitHub** markdown format, and I'd like to convert these to Microsoft Word format (I have tested other output formats as well but it seems that there's no support for this in the input format), but I have noticed that this is not currently possible using pandoc. I would really like this to work as that would make pandoc just that much better in my opinion.

I'm using pandoc version 2.10.1 but I haven't seen any math/formula support added in any of the versions named in the changelog so I doubt that matters much.

Thanks in advance!

**Comment by tarleb** [comment]
Can you provide an example? I was under the impression that gfm does not support formulas.

Either way, you can tell pandoc to use a modified gfm which can deal with formulas by enabling the `tex_math_dollars` extension (and other math-related extensions):

    pandoc --from=gfm+tex_math_dollars

**Comment by LucAngevare** [comment]
Apologies, apparently I didn't do enough research.
I did not know that GitHub markdown didn't support equations and formulas, I thought they supported MathJax which meant that equations would be rendered as images and those would display inside the markdown as the formula, which apparently is not the case. I also did not read far enough into the docs (I only read the index, about, installing and getting started pages) to know that adding `+tex_math_dollars` would make it so equations are supported, apologies for wasting your time and thanks for your response/help. Closing this for these reasons.
